src/demos/demo1.ts

```diff
@@ -112,32 +118,47 @@ function main() {
         .setChannel(1)
         .setMountFace(RobotSpecs.SensorMountingFace.REAR)
     )
+    .addBasicSensor(
+      distanceSensor
+        .copy()
+        .setChannel(2)
+        .setMountFace(RobotSpecs.SensorMountingFace.RIGHT)
+    )
     .addWheel("left-drive", wheel)
     .addWheel(
       "left-drive",
       wheel
         .copy()
         .setMountPoint(RobotSpecs.WheelMountingPoint.LEFT_REAR)
-        .setMountOffset({ x: -0.075, y: -0.25, z: -0.5 })
+        .setMountOffset({ x: -0.075, y: -0.25, z: -0.85 })
     )
     .addWheel(
       "right-drive",
       wheel
         .copy()
         .setMountPoint(RobotSpecs.WheelMountingPoint.RIGHT_FRONT)
-        .setMountOffset({ x: 0.075, y: -0.25, z: 0.5 })
+        .setMountOffset({ x: 0.075, y: -0.25, z: 0.7 })
     )
     .addWheel(
       "right-drive",
       wheel
         .copy()
         .setMountPoint(RobotSpecs.WheelMountingPoint.RIGHT_REAR)
-        .setMountOffset({ x: 0.075, y: -0.25, z: -0.5 })
+        .setMountOffset({ x: 0.075, y: -0.25, z: -0.85 })
     )
     .addMotor("left-drive", motor)
     .addMotor("right-drive", motor.copy().setChannel(1));
 
-  const robot = simulator.addRobot(robotBuilder.generateSpec());
+  const spec = robotBuilder.generateSpec();
+  spec.customMesh = {
+    filePath: "assets/models/robot.gltf",
+    rotation: {
+      x: 0,
+      y: Math.PI / 2,
```

Comment:
Why Math.PI specifically?

Reply:
I would guess the rotations are described in radians instead of degrees, with `Math.PI` radians representing 180 degrees.

Reply:
Yep, everything in radians